packages/terra-application-header-layout/docs/README.md

```diff
@@ -0,0 +1,29 @@
+# Terra Application Header Layout
+
+This component renders an application header layout. To be used with a terra-layout or terra-navigation-layout.
+
+## Getting Started
+
+- Install with [npmjs](https://www.npmjs.com):
+  - `npm install terra-application-header-layout`
+  - `yarn add terra-application-header-layout`
+
+## Usage
+
+```jsx
+import React from 'react';
+import ApplicationHeaderLayout from 'terra-application-header-layout';
+
+<ApplicationHeaderLayout
```

Comment:
Should probably include example that uses the layoutProps that would be paired with the toggle element.

Reply:
Removed layoutProps after Tyler's input, so it shouldn't be needed now.